## 71% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 15
Expected to fail tests: 0
Passed tests: 36
```
```
IDs of failed tests: 702840,702894,703086,702947,702844,704613,702841,702839,702809,702838,704615,702859,702957,702843,703194 
```
<h3>Failed tests (15)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/block-user-from-editing-name-with-ens&PR_ID=21003&APK_NAME=21003.apk&TR_CASE_IDS=702840,702894,703086,702947,702844,704613,702841,702839,702809,702838,704615,702859,702957,702843,703194">Rerun failed tests</a></li><br/><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655437">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a message by text: https://status.app/u/CweACg0KC1Rlc3RVc2VyRTJFAw==#zQ3shcFXYnGXxJZnsMThziUNMwyA5uGLp58bLGmfb3qaWD1F6 </div><div>Device 1: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/u/CweACg0KC1Rlc3RVc2VyRTJFAw==#zQ3shcFXYnGXxJZnsMThziUNMwyA5uGLp58bLGmfb3qaWD1F6')]` </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:41: in test_links_open_universal_links_from_chat
    self.channel.chat_element_by_text(url).click_on_link_inside_message_body()
../views/chat_view.py:176: in click_on_link_inside_message_body
    self.message_body.click_inside_element_by_coordinate(rel_x=0.1, rel_y=0.9)
../views/base_element.py:357: in click_inside_element_by_coordinate
    location, size = self.get_element_coordinates()
../views/base_element.py:296: in get_element_coordinates
    size = element.size
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:275: in size
    size = self._execute(Command.GET_ELEMENT_RECT)["value"]
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webelement.py:395: in _execute
    return self._parent.execute(command, params)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs-rerun@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 The element 'By.xpath: //android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/u/CweACg0KC1Rlc3RVc2VyRTJFAw==#zQ3shcFXYnGXxJZnsMThziUNMwyA5uGLp58bLGmfb3qaWD1F6')]' is not linked to the same object in DOM anymore; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#stale-element-reference-exception
E   Stacktrace:
E   io.appium.uiautomator2.common.exceptions.StaleElementReferenceException: The element 'By.xpath: //android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'https://status.app/u/CweACg0KC1Rlc3RVc2VyRTJFAw==#zQ3shcFXYnGXxJZnsMThziUNMwyA5uGLp58bLGmfb3qaWD1F6')]' is not linked to the same object in DOM anymore
E   	at io.appium.uiautomator2.model.ElementsCache.restore(ElementsCache.java:122)
E   	at io.appium.uiautomator2.model.ElementsCache.get(ElementsCache.java:153)
E   	at io.appium.uiautomator2.handler.GetRect.safeHandle(GetRect.java:40)
E   	at io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:59)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:277)
E   	at io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:271)
E   	at io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:435)
E   	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:293)
E   	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:267)
E   	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:250)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:266)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:345)
E   	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1294)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)
E   	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)
E   	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:911)
E   	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:131)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:611)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:552)
E   	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:466)
E   	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:438)
E   	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:140)
E   	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:144)
E   	at java.lang.Thread.run(Thread.java:1012)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94b9e5eb0c9e4c889a255c2047294ae9?auth=3929e3922c5114d42577805140007806#61">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655422">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='user-avatar']/following-sibling::android.widget.TextView[@text='user2']` </div></blockquote></p>

```
Test setup failed: activity_center/test_activity_center.py:270: in prepare_devices
    self.community_1.invite_to_community(self.community_name, self.username_2)
../views/chat_view.py:486: in invite_to_community
    Button(self.driver, xpath=xpath).click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by xpath: `//*[@content-desc='user-avatar']/following-sibling::android.widget.TextView[@text='user2']` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b7d9182e6ed74679bd7f87a424bba483?auth=b180cfccc5315ad98de928aa434b2840">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/89fc76e250924a38825e552655876077?auth=c43220d9ec4e48a1b3ff394b183ab8e2">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655424">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: SendMessageButton </div><div># STEP: Checking unread indicators</div></blockquote></p>

```
activity_center/test_activity_center.py:428: in test_activity_center_mentions
    self.home_1.notifications_unread_badge.wait_for_visibility_of_element(120)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: BaseElement by accessibility id:`activity-center-unread-count` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8f4fe710b69247c49b31a0ba56c4c05e?auth=c392865cf01288d2144f25058a264b55#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/893b2ca8e279437fa49761bc16d8167e?auth=c97ea4d204bef052886a279fee452a3c#101">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655406">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for a message by text: quote reply (one row) </div><div>Device 2: Looking for a message by text: quote reply (one row) </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1009: in test_community_markdown_support
    self.errors.verify_no_errors()
base_test_case.py:191: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 code blocks is not displayed with markdown in community channel for the recipient (device 1)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aa4b7a61dc454a4e8545fe44e60d99a1?auth=fa58d071efa18e712acaf050f622f1f5#194">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d81475db31a94e8aa78b644b34de47ad?auth=0a6102e8c7d0adb854a6e655601d4cbe#174">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655410">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655421">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655426">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655413">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655411">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655409">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655408">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: Button </div><div>Device 2: Attempt 0 is successful clicking close-activity-center </div></blockquote></p>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f98f176f24e846bd90780335be32500e?auth=1e303c492d5cedaf3ec5ea7ebfb0bf26">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/166da419625242e4999296b422c6650e?auth=44836d81bae5286a58979e2b366d46df">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655438">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655419">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655412">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655428">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:320: in prepare_devices
    self.home_2.handle_contact_request(self.username_1)
../views/home_view.py:409: in handle_contact_request
    chat_element.accept_contact_request()
../views/home_view.py:162: in accept_contact_request
    self.handle_cr("accept-contact-request")
../views/home_view.py:159: in handle_cr
    ).wait_for_rendering_ended_and_click()
../views/base_element.py:163: in wait_for_rendering_ended_and_click
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: Button by xpath:`//*[contains(@text, 'user_1')]/ancestor::*[@content-desc='activity']/*[@content-desc="accept-contact-request"]` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/></td></tr></tbody></table></details><h3>Passed tests (36)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655396">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dcffc7cdde194254b6871810f7bd9628?auth=ccf915d3dbe98bbc7fb9a840bdb1e518#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655415">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dcffc7cdde194254b6871810f7bd9628?auth=ccf915d3dbe98bbc7fb9a840bdb1e518#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655427">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dcffc7cdde194254b6871810f7bd9628?auth=ccf915d3dbe98bbc7fb9a840bdb1e518#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655420">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dcffc7cdde194254b6871810f7bd9628?auth=ccf915d3dbe98bbc7fb9a840bdb1e518#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655431">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dcffc7cdde194254b6871810f7bd9628?auth=ccf915d3dbe98bbc7fb9a840bdb1e518#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655435">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/dcffc7cdde194254b6871810f7bd9628?auth=ccf915d3dbe98bbc7fb9a840bdb1e518#355">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655440">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655439">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655398">test_links_deep_links_profile</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94b9e5eb0c9e4c889a255c2047294ae9?auth=3929e3922c5114d42577805140007806#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655442">test_deep_links_communities</a>, id: 739307 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94b9e5eb0c9e4c889a255c2047294ae9?auth=3929e3922c5114d42577805140007806#187">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655434">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b9c00df729124cbd9564ea8aaedab7b4?auth=cdb4288eb95b9163b5ff778e03a3d6d5#111">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4876fa01ec31439393bc89725e3a1c39?auth=93743a98160e96b6f41c644b9bca3ff0#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655401">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b9c00df729124cbd9564ea8aaedab7b4?auth=cdb4288eb95b9163b5ff778e03a3d6d5#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4876fa01ec31439393bc89725e3a1c39?auth=93743a98160e96b6f41c644b9bca3ff0#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655402">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b9c00df729124cbd9564ea8aaedab7b4?auth=cdb4288eb95b9163b5ff778e03a3d6d5#177">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4876fa01ec31439393bc89725e3a1c39?auth=93743a98160e96b6f41c644b9bca3ff0#131">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655418">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9f550f6f75bb4a6293205d1e7d1fa698?auth=6c565334b60643bbe88ceb3156bdfae1#464">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/43468f3c34b047e2bb6301a59a961aae?auth=642b29de1fa9dd167cc4431acc70c513#296">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655392">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9f550f6f75bb4a6293205d1e7d1fa698?auth=6c565334b60643bbe88ceb3156bdfae1#101">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/43468f3c34b047e2bb6301a59a961aae?auth=642b29de1fa9dd167cc4431acc70c513#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655397">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9f550f6f75bb4a6293205d1e7d1fa698?auth=6c565334b60643bbe88ceb3156bdfae1#357">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/43468f3c34b047e2bb6301a59a961aae?auth=642b29de1fa9dd167cc4431acc70c513#215">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655393">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9f550f6f75bb4a6293205d1e7d1fa698?auth=6c565334b60643bbe88ceb3156bdfae1#219">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/43468f3c34b047e2bb6301a59a961aae?auth=642b29de1fa9dd167cc4431acc70c513#170">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655395">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9f550f6f75bb4a6293205d1e7d1fa698?auth=6c565334b60643bbe88ceb3156bdfae1#556">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/43468f3c34b047e2bb6301a59a961aae?auth=642b29de1fa9dd167cc4431acc70c513#402">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655407">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9f550f6f75bb4a6293205d1e7d1fa698?auth=6c565334b60643bbe88ceb3156bdfae1#415">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/43468f3c34b047e2bb6301a59a961aae?auth=642b29de1fa9dd167cc4431acc70c513#265">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655400">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9f550f6f75bb4a6293205d1e7d1fa698?auth=6c565334b60643bbe88ceb3156bdfae1#153">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/43468f3c34b047e2bb6301a59a961aae?auth=642b29de1fa9dd167cc4431acc70c513#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655432">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9f550f6f75bb4a6293205d1e7d1fa698?auth=6c565334b60643bbe88ceb3156bdfae1#495">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/43468f3c34b047e2bb6301a59a961aae?auth=642b29de1fa9dd167cc4431acc70c513#342">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655429">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b5552316e9646f8ac20863f8e388104?auth=e8429a52580520fc61605c03ac621e4f#187">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2351fbb356af4b0bb6804070a2c2e57c?auth=e7493f3a9ff611f61d96878e381a239f#112">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/104e3adcd91b4687b1063493414cd230?auth=b589c5c4e98aec2f68bc829239b6f11a#92">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655404">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b5552316e9646f8ac20863f8e388104?auth=e8429a52580520fc61605c03ac621e4f#170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2351fbb356af4b0bb6804070a2c2e57c?auth=e7493f3a9ff611f61d96878e381a239f#84">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/104e3adcd91b4687b1063493414cd230?auth=b589c5c4e98aec2f68bc829239b6f11a#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655405">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b5552316e9646f8ac20863f8e388104?auth=e8429a52580520fc61605c03ac621e4f#335">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2351fbb356af4b0bb6804070a2c2e57c?auth=e7493f3a9ff611f61d96878e381a239f#211">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/104e3adcd91b4687b1063493414cd230?auth=b589c5c4e98aec2f68bc829239b6f11a#207">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655394">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b5552316e9646f8ac20863f8e388104?auth=e8429a52580520fc61605c03ac621e4f#368">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2351fbb356af4b0bb6804070a2c2e57c?auth=e7493f3a9ff611f61d96878e381a239f#238">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/104e3adcd91b4687b1063493414cd230?auth=b589c5c4e98aec2f68bc829239b6f11a#234">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655430">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b5552316e9646f8ac20863f8e388104?auth=e8429a52580520fc61605c03ac621e4f#294">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2351fbb356af4b0bb6804070a2c2e57c?auth=e7493f3a9ff611f61d96878e381a239f#176">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/104e3adcd91b4687b1063493414cd230?auth=b589c5c4e98aec2f68bc829239b6f11a#147">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655433">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7b5552316e9646f8ac20863f8e388104?auth=e8429a52580520fc61605c03ac621e4f#488">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/2351fbb356af4b0bb6804070a2c2e57c?auth=e7493f3a9ff611f61d96878e381a239f#281">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/104e3adcd91b4687b1063493414cd230?auth=b589c5c4e98aec2f68bc829239b6f11a#248">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655417">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7cddf85eb5324ea98df0b6c285497a84?auth=038aa860cacc66942b32ac9ced3a3e73#105">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/15f3b2088b244d06a2a77e062d849aeb?auth=b571c4d53ccb3881d51f960c77e7d1f0#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655416">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7cddf85eb5324ea98df0b6c285497a84?auth=038aa860cacc66942b32ac9ced3a3e73#62">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/15f3b2088b244d06a2a77e062d849aeb?auth=b571c4d53ccb3881d51f960c77e7d1f0#56">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655399">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7cddf85eb5324ea98df0b6c285497a84?auth=038aa860cacc66942b32ac9ced3a3e73#140">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/15f3b2088b244d06a2a77e062d849aeb?auth=b571c4d53ccb3881d51f960c77e7d1f0#206">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655425">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8f4fe710b69247c49b31a0ba56c4c05e?auth=c392865cf01288d2144f25058a264b55#166">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/893b2ca8e279437fa49761bc16d8167e?auth=c97ea4d204bef052886a279fee452a3c#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655414">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aa4b7a61dc454a4e8545fe44e60d99a1?auth=fa58d071efa18e712acaf050f622f1f5#309">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d81475db31a94e8aa78b644b34de47ad?auth=0a6102e8c7d0adb854a6e655601d4cbe#276">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655403">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aa4b7a61dc454a4e8545fe44e60d99a1?auth=fa58d071efa18e712acaf050f622f1f5#145">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d81475db31a94e8aa78b644b34de47ad?auth=0a6102e8c7d0adb854a6e655601d4cbe#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655423">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aa4b7a61dc454a4e8545fe44e60d99a1?auth=fa58d071efa18e712acaf050f622f1f5#312">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d81475db31a94e8aa78b644b34de47ad?auth=0a6102e8c7d0adb854a6e655601d4cbe#298">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655436">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/aa4b7a61dc454a4e8545fe44e60d99a1?auth=fa58d071efa18e712acaf050f622f1f5#511">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d81475db31a94e8aa78b644b34de47ad?auth=0a6102e8c7d0adb854a6e655601d4cbe#483">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3655441">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9cc5a68c416847fa9d7f23eb0f456423?auth=10cd739f8ff029c5714d39d6d7fe9d47#39">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/legacy/status_im/data_store/settings.cljs
+++ b/src/legacy/status_im/data_store/settings.cljs
@@ -1,9 +1,10 @@
 (ns legacy.status-im.data-store.settings
   (:require
-    [clojure.set :as set]
-    [clojure.string :as string]
-    [legacy.status-im.data-store.visibility-status-updates :as visibility-status-updates]
-    [utils.ethereum.eip.eip55 :as eip55]))
+   [clojure.set :as set]
+   [clojure.string :as string]
+   [legacy.status-im.data-store.visibility-status-updates :as visibility-status-updates]
+   [status-im.contexts.wallet.common.validation :as validation]
+   [utils.ethereum.eip.eip55 :as eip55]))
 
 (defn rpc->visible-tokens
   [visible-tokens]
@@ -35,6 +36,7 @@
       (update :pinned-mailservers rpc->pinned-mailservers)
       (update :link-previews-enabled-sites set)
       (update :currency rpc->currency)
+      (assoc :ens-name? (validation/ens-name? (:preferred-name settings)))
       (visibility-status-updates/<-rpc-settings)
       (set/rename-keys {:compressedKey :compressed-key
                         :emojiHash     :emoji-hash})))

--- a/src/status_im/common/universal_links.cljs
+++ b/src/status_im/common/universal_links.cljs
@@ -178,13 +178,10 @@
   [{:keys [db]} [{:keys [public-key on-success]}]]
   (let [profile-public-key (get-in db [:profile/profile :public-key])
         profile?           (or (not public-key) (= public-key profile-public-key))
-        ens-name?          (if profile?
-                             (get-in db [:profile/profile :ens-name?])
-                             (get-in db [:contacts/contacts public-key :ens-name]))
         public-key         (if profile? profile-public-key public-key)]
     (when public-key
       {:json-rpc/call
-       [{:method     (if ens-name? "wakuext_shareUserURLWithENS" "wakuext_shareUserURLWithData")
+       [{:method     "wakuext_shareUserURLWithData"
          :params     [public-key]
          :on-success (fn [url]
                        (rf/dispatch [:universal-links/save-profile-url public-key url])

--- a/src/status_im/common/universal_links_test.cljs
+++ b/src/status_im/common/universal_links_test.cljs
@@ -45,8 +45,8 @@
       (let [db  {:profile/profile {:ens-name? true :public-key pubkey}}
             rst (links/generate-profile-url {:db db} [])]
         (are [result expected] (match? result expected)
-         "wakuext_shareUserURLWithENS" (-> rst :json-rpc/call first :method)
-         pubkey                        (-> rst :json-rpc/call first :params first)))))
+         "wakuext_shareUserURLWithData" (-> rst :json-rpc/call first :method)
+         pubkey                         (-> rst :json-rpc/call first :params first)))))
   (testing "user has no ens name"
     (testing "it calls the ens rpc method with public keyas param"
       (let [db  {:profile/profile {:public-key pubkey}}
@@ -60,8 +60,8 @@
             db  {:contacts/contacts {pubkey {:ens-name ens}}}
             rst (links/generate-profile-url {:db db} [{:public-key pubkey}])]
         (are [result expected] (match? result expected)
-         "wakuext_shareUserURLWithENS" (-> rst :json-rpc/call first :method)
-         pubkey                        (-> rst :json-rpc/call first :params first)))))
+         "wakuext_shareUserURLWithData" (-> rst :json-rpc/call first :method)
+         pubkey                         (-> rst :json-rpc/call first :params first)))))
   (testing "contact has no ens name"
     (testing "it calls the ens rpc method with public keyas param"
       (let [db  {:contacts/contacts {pubkey {:public-key pubkey}}}

--- a/src/status_im/contexts/profile/edit/list_items.cljs
+++ b/src/status_im/contexts/profile/edit/list_items.cljs
@@ -11,18 +11,18 @@
 
 (defn items
   [theme]
-  (let [profile             (rf/sub [:profile/profile-with-image])
-        customization-color (rf/sub [:profile/customization-color])
-        bio                 (:bio profile)
-        full-name           (profile.utils/displayed-name profile)]
+  (let [{:keys [bio ens-name?] :as profile} (rf/sub [:profile/profile-with-image])
+        customization-color                 (rf/sub [:profile/customization-color])
+        full-name                           (profile.utils/displayed-name profile)]
     [{:label (i18n/label :t/profile)
-      :items [{:title           (i18n/label :t/name)
-               :on-press        #(rf/dispatch [:open-modal :edit-name])
-               :blur?           true
-               :label           :text
-               :label-props     (utils/truncate-str full-name constants/profile-name-max-length)
-               :action          :arrow
-               :container-style style/item-container}
+      :items [(cond-> {:title           (i18n/label :t/name)
+                       :blur?           true
+                       :label           :text
+                       :label-props     (utils/truncate-str full-name constants/profile-name-max-length)
+                       :container-style style/item-container}
+                (not ens-name?)
+                (assoc :on-press #(rf/dispatch [:open-modal :edit-name])
+                       :action   :arrow))
               {:title           (i18n/label :t/bio)
                :on-press        #(rf/dispatch [:open-modal :edit-bio])
                :blur?           true

--- a/src/status_im/contexts/wallet/common/validation.cljs
+++ b/src/status_im/contexts/wallet/common/validation.cljs
@@ -2,7 +2,10 @@
   (:require [clojure.string :as string]
             [status-im.constants :as constants]))
 
-(defn ens-name? [s] (if (string/blank? s) false (boolean (re-find constants/regx-ens s))))
+(defn ens-name?
+  [s]
+  (and (not (string/blank? s))
+       (boolean (re-find constants/regx-ens s))))
 (defn private-key?
   [s]
   (or (re-find constants/regx-private-key-hex s)

--- a/src/status_im/contexts/wallet/common/validation.cljs
+++ b/src/status_im/contexts/wallet/common/validation.cljs
@@ -1,7 +1,8 @@
 (ns status-im.contexts.wallet.common.validation
-  (:require [status-im.constants :as constants]))
+  (:require [clojure.string :as string]
+            [status-im.constants :as constants]))
 
-(defn ens-name? [s] (boolean (re-find constants/regx-ens s)))
+(defn ens-name? [s] (if (string/blank? s) false (boolean (re-find constants/regx-ens s))))
 (defn private-key?
   [s]
   (or (re-find constants/regx-private-key-hex s)

